ParisBNF/abb/BNFabb105.xml

```diff
@@ -0,0 +1,979 @@
+                           </explicit>
+                           </msItem>
+
+                           <msItem xml:id="ms_i1.3">
+                             <locus from="122r" to="127v"/>
+                             <title type="complete" ref="LIT2362Songof"><hi rend="rubric">ማኃልየ፡ መኃልይ᎓ ዘውእቱ᎓ ዘሰሎሞን፡ ንጉሥ።</hi></title>
+                             <textLang mainLang="gez"/>
+                             <incipit xml:lang="gez"><hi rend="rubric">፩</hi>ይስዕመኒ᎓ በስዕመተ᎓ አፉሁ፨<lb/>
+                          አዳም᎓ አጥባተኪ᎓ እምወይን፨<lb/>
+                        </incipit>
+                            <explicit xml:lang="gez">ጕይይ᎓ አንተ፡ ወልድ᎓ እኁየ᎓ ወተመሰለ᎓ ለወይጠል፨<lb/>
+                            ወእመ᎓ አኮ᎓ ከመ᎓ ወሬዛ᎓ ሀየል። ። ። ። ።<lb/></explicit>
+                           </msItem>
+
+                           <msItem xml:id="ms_i1.4">
+                             <locus from="128r" to="155v"/>
+                             <title type="complete" ref="LIT1961Mashaf">Daily prayers</title>
+                             <textLang mainLang="gez"/>
+                             <incipit xml:lang="gez"><hi rend="rubric">በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ።
+                               አሐ</hi>ዱ፡ አምላክ፨ ንወጥን፡ በፈሪሀ፡ እግዚአብሔር፨ <hi rend="rubric">ሎቱ፡ ስብሐተ፡ በእንተ፡ ጽሒፈ፡ ሰዐታተ፡ ወጸሎታ</hi>ተ፡
+                               ዘመዓልተ፡ ወለሊተ፡
+                               ወውእቱ፡ ረዳኢነ፡ ምስለ፡ ገቢ<hi rend="rubric">ር፡ ለፈጽሞ፡ ዘነገሮ፡ መልአከ፡ እግዚአብሔር፡ ለ</hi>አባ፡ እንጦንስ፡
+                             ጸሎቱ፡ ተሀሎ፡ ምስሌነ፡ አሜን፨ </incipit>
+                            <explicit xml:lang="gez">ይስረይ᎓ ኃጢአተነ᎓ በብዝኃ᎓ ምሕረቱ᎓ ሰአሊ᎓ ለነ᎓ ቅድስት፨፨፨</explicit>
+                           </msItem>
+
+                           <msItem xml:id="ms_i1.5">
+                             <locus from="157r" to="181v"/>
+                             <title type="complete" ref="LIT2509Weddas"/>
+                             <textLang mainLang="gez"/>
+                             <incipit xml:lang="gez"><hi rend="rubric">በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ። አ</hi>
+                             ሐዱ᎓ አምላክ፨ ወእምዝ፡ ንጽሕፍ፡ ው<hi rend="rubric">ዳሴሃ᎓ ለእግዚእትነ᎓ ማርያም᎓ ድንግል፡ ወላ</hi>ዲተ᎓ አምላክ᎓ <gap reason="ellipsis" resp="DN"/>
+                             ተሰመይኪ᎓ ፍቅርተ፡ ኦቡርክት፡ እምአንስት፡ አንቲ፡ ይእቲ፡ ዳግሚት፡ ቀመር፡ እንተ፡ ተሰመይኪ፡ ቅድስተ፡ ቅዱሳን። ዘውስቴታ፡ ጽላተ፡ ኪዳን፡</incipit>
+                             <explicit xml:lang="gez">
+                             ኦእግዚእትነ᎓ እሙ᎓ ለዐቢይ፡ ወክቡር፡ አሐዜ፡ ኵሉ፡ ዓለም፡ ዘምሉእ፡ ሰማያ፡ ተ፡ ወምድር፡ ስብሐቲሁ፡ ሎቱ፡ ንገኒ፡ ወሎቱ፡ ነአምን፡
+                             እምይእዜ፡ ወእስከ፡ ለዓለም፡ ዓለም፡ አሜን፨ <hi rend="rubric">
+                             ሠረቀ᎓ በሥጋ᎓ እምድንግል᎓ ዘእን</hi></explicit>
+                           </msItem>
+
+                           <msItem xml:id="ms_i1.6">
+                             <locus from="181v" to="186ra"/>
+                             <title type="complete" ref="LIT4711Mastabqwe"/>
+                             <textLang mainLang="gez"/>
+                               <incipit xml:lang="gez"><hi rend="rubric">ውዳሴ᎓ ወግናይ᎓ ዘእግዚእትነ᎓ ማርያም᎓᎓</hi> ወላዲተ᎓
```

Comment:
in LIT4711Mastabqwe a part is commented out (I don't know why, but maybe @thea-m knows) but once it is fixed this msItem should get the ref="LIT4711Mastabqwe#Maryam" if I see it right

Reply:
@eu-genia thank you for the reminder! I have a document with more text transcribed by Leo, but not completely added to the file. There were some uncertainties regarding the subdivision. I'll look at it again now and will write an issue to remind myself if I don't finish it today.

Reply:
No, this is a mistake, this is a general id.

Reply:
@DenisNosnitsin1970 don't understand, check https://betamasaheft.eu/LIT4711Mastabqwe.xml the commented part below

Reply:
What exactly I do not understand, or what exactly is problematic? If you mean <incipit xml:lang="gez"><hi rend="rubric">ውዳሴ᎓ ወግናይ᎓ ዘእግዚእትነ᎓ ማርያም᎓᎓, this part is gone. There are still a couple of supplications without refs, but Dorothea says she has some more supplications that have been transcribed by Leo, not yet included under  LIT4711Mastabqwe. Obviously I have to wait untill everything has been included, and then have another look.